src/app/modules/shared/components/navbar/navbar.component.html

```diff
@@ -56,16 +56,23 @@
             >My Deposits</a
           >
         </li>
+
         <li *ngIf="!authenticated" class="nav-item">
-          <a class="btn-auth" routerLink="/register" routerLinkActive="active"
-            >Sign up</a
-          >
-        </li>
-        <li *ngIf="!authenticated" class="nav-item">
-          <a class="btn-auth" routerLink="/login" routerLinkActive="active"
+          <a class="nav-link waves-effect waves-light" routerLink="/login" routerLinkActive="active"
             >Log in
           </a>
         </li>
+        <button
+          *ngIf="!authenticated"
+          class="btn-auth"
+          routerLink="/register"
+          routerLinkActive="active"
+        >
+          Sign up
+        </button>
+<!--        <button class="btn" type="submit">-->
+<!--          Log in-->
+<!--        </button>-->
```

Comment:
Remember to remove the comments, if they no longer used

Reply:
i have worked on this